akka-docs-dev/rst/scala/http/routing-dsl/directives/basic-directives/cancelRejections.rst

```diff
@@ -0,0 +1,23 @@
+.. _-cancelRejections-:
+
+cancelRejections
+================
+
+...
+
+Signature
+---------
+
+.. includecode2:: /../../akka-http-scala/src/main/scala/akka/http/scaladsl/server/directives/BasicDirectives.scala
+   :snippet: cancelRejections
+
+Description
+-----------
+
+...
+
+Example
+-------
+
+... includecode2:: ../../../../code/docs/http/scaladsl/server/directives/BasicDirectivesExamplesSpec.scala
+   :snippet: 0cancelRejections
```

Comment:
It was used when the name of one directive is a prefix of another one. Our `includeCode` macro gets easily confused in that case.